src/py42/services/storage/exfiltrateddata.py

```diff
@@ -1,37 +1,25 @@
-from urllib.parse import quote
-
 from py42.services import BaseService
 
 
 class ExfiltratedDataService(BaseService):
 
     _base_uri = "api/v1/"
 
-    def __init__(self, main_session, streaming_session):
-        super().__init__(main_session)
+    def __init__(self, main_connection, streaming_session):
+        super().__init__(main_connection)
         self._streaming_session = streaming_session
 
-    def get_download_token(
-        self, event_id, device_id, file_path, file_sha256, timestamp
-    ):
+    def get_download_token(self, downloadRequestUrl):
```

Comment:
Is this the only customer using this as far as we know? Changing the signature is technically a breaking change - but if its currently broken and/or no ones using it, then its probably okay.  
Maybe we could add details about how to adopt the fix in the changelog?

Reply:
Added a note to the changelog.